src/ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -125,6 +125,7 @@ add_library(onnx_import STATIC
         op/pow.hpp
         op/prelu.cpp
         op/prelu.hpp
+        op/qmatmul.hpp
```

Comment:
Please preserve style: the other quantized ops functions are named with `quant` or `quantize` (_full_) prefix.